types/zenscroll/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for zenscroll 4.0.0
```

Comment:
Please add dtslint.json to this PR as requested in the template

Reply:
Should have a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. (not `dtslint.json`)

Reply:
I added it no problem now